index.html

```diff
@@ -7,32 +7,54 @@
     <meta name="description" content="">
     <meta name="author" content="">
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="style.css">
```

Comment:
You'll need to serve this file as well. Right now it only serves the index & each of the components. Not set up to serve `.css` files yet 👍